raiden/network/matrixtransport.py

```diff
@@ -72,7 +72,7 @@ class MatrixTransport:
     def __init__(self, config: dict):
         self._raiden_service: RaidenService = None
         self._server_url: str = self._select_server(config)
-        self._server_name = config.get('server_name', urlparse(self._server_url).hostname)
+        self._server_name = config.get('server_name', urlparse(self._server_url).netloc)
```

Comment:
Using `hostname` instead of `netloc` was intentional here since `_server_name` is also used to construct fully qualified room names which will fail if it includes a port.